.github/stale.yml

```diff
@@ -0,0 +1,31 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 60
```

Comment:
In my opinion this is the most reasonable - one month is too short, 3 months kinda too long?